tests/sdk/test_config.py

```diff
@@ -32,6 +34,49 @@ def test_add_load_profile(tmpdir):
     assert profile_1 == profile_2
 
 
+def test_add_load_profile_from_file(tmpdir):
+    path = tmpdir / 'substra.cfg'
+    conf = {
+       "node-1": {
+           "auth": {
+               "username": "node-1"
+           },
+           "insecure": False,
+           "url": "http://substra-backend.node-1.com",
+           "version": "0.0"
+       },
+    }
+
+    path.write_text(json.dumps(conf), "UTF-8")
+    keyring.set_password('node-1', conf['node-1']['auth']['username'], 'foo')
+    manager = configuration.Manager(str(path))
+    profile = manager.load_profile('node-1')
+
+    assert conf['node-1']['auth']['username'] == profile['auth']['username']
+    assert keyring.get_password(service_name='node-1', username='node-1') == 'foo'
```

Comment:
The keyring.set_password, keyring.get_password only test that keyring indeed works, but doesn't check anything from our code. When we're using keyring, we're assuming that it's been properly tested by its developers and don't need to re-test it.

Reply:
Alright, so I should remove this link?

Reply:
I think so, yes.